packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianKeyValues.tsx

```diff
@@ -23,20 +23,9 @@ import { KeyValuePair, Link } from '../../../../types/trace';
 
 import './AccordianKeyValues.css';
 
-type AccordianKeyValuesProps = {
-  className?: string | TNil;
-  data: KeyValuePair[];
-  highContrast?: boolean;
-  interactive?: boolean;
-  isOpen: boolean;
-  label: string;
-  linksGetter: ((pairs: KeyValuePair[], index: number) => Link[]) | TNil;
-  onToggle?: null | (() => void);
-};
-
 // export for tests
 export function KeyValuesSummary(props: { data?: KeyValuePair[] }) {
-  const { data } = props;
+  const { data } = props || null;
```

Comment:
don't follow this. If props are empty you replace them with null? How then can you deconstruct null into `{ data }` object?

Reply:
> `props || {}`

why do you need even that? `props` are not nullable in the signature, so they are already required to be at least `{}`